CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG ${CMAKE_CXX_FLAGS_DEBUG}")
 if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR
         CMAKE_CXX_COMPILER_ID MATCHES ".*Clang")
     set(WARNING_FLAGS "-Wall -Wextra -Wno-unused-parameter")
-    set(CMAKE_CXX_FLAGS "-std=c++0x ${WARNING_FLAGS} ${CMAKE_CXX_FLAGS}")
+    set(CMAKE_CXX_FLAGS "-std=gnu++11 ${WARNING_FLAGS} ${CMAKE_CXX_FLAGS}")
```

Comment:
I'm curious what problem this solved...  I'd prefer to stick to standard C++ with no GNU extensions if possible.  Also, c++0x should be a synonym for c++11, but the latter is only supported by GCC 4.7 or later (we're currently targetting GCC 4.6 minimum)